tests/integration/watcher/certificate_test.go

```diff
@@ -33,34 +36,20 @@ var _ = Describe("Create Watcher Certificates", Ordered, func() {
 				ObjectMeta: apimetav1.ObjectMeta{
 					Name:        "test-kyma-1",
 					Namespace:   "testcase-1",
-					Annotations: map[string]string{watcher.DomainAnnotation: "example.domain.com"},
+					Annotations: map[string]string{shared.SKRDomainAnnotation: "example.domain.com"},
 				},
 			},
 			issuer:         NewTestIssuer("testcase-1"),
 			wantCreateErr:  false,
 			wantNewCertErr: false,
 		},
-		{
-			name:      "Should fail since no Issuer can be found",
-			namespace: NewTestNamespace("testcase-2"),
-			kyma: &v1beta2.Kyma{
-				ObjectMeta: apimetav1.ObjectMeta{
-					Name:        "test-kyma-2",
-					Namespace:   "testcase-2",
-					Annotations: map[string]string{watcher.DomainAnnotation: "example.domain.com"},
-				},
-			},
-			issuer:         nil,
-			wantCreateErr:  true,
-			wantNewCertErr: false,
-		},
```

Comment:
This test is not needed anymore because we don't lookup the Issuer by label but configure its name via flag.